#--- libdc1394-0.9.5/examples/Makefile.in.lib64	2004-06-30 03:25:27.000000000 -0400
#+++ libdc1394-0.9.5/examples/Makefile.in	2004-09-20 05:29:04.357518841 -0400
#@@ -99,7 +99,7 @@ grab_partial_image_SOURCES = grab_partia
# grab_partial_image_LDADD = ../libdc1394/libdc1394_control.la -lraw1394
# 
# dc1394_multiview_SOURCES = dc1394_multiview.c
#-dc1394_multiview_LDADD = ../libdc1394/libdc1394_control.la -lraw1394 -L/usr/X11R6/lib -lXv -lXext -lX11 
#+dc1394_multiview_LDADD = ../libdc1394/libdc1394_control.la -lraw1394 @X_LIBS@ -lXv -lXext -lX11 
# 
# dc1394_vloopback_SOURCES = dc1394_vloopback.c affine.c affine.h
# dc1394_vloopback_LDADD = -lm ../libdc1394/libdc1394_control.la -lraw1394
#--- libdc1394-0.9.5/examples/Makefile.am.lib64	2003-09-15 13:21:27.000000000 -0400
#+++ libdc1394-0.9.5/examples/Makefile.am	2004-09-20 05:28:59.186475730 -0400
#@@ -12,7 +12,7 @@ grab_partial_image_SOURCES = grab_partia
# grab_partial_image_LDADD = ../libdc1394/libdc1394_control.la -lraw1394
# 
# dc1394_multiview_SOURCES = dc1394_multiview.c
#-dc1394_multiview_LDADD = ../libdc1394/libdc1394_control.la -lraw1394 -L/usr/X11R6/lib -lXv -lXext -lX11 
#+dc1394_multiview_LDADD = ../libdc1394/libdc1394_control.la -lraw1394 @X_LIBS@ -lXv -lXext -lX11 
# 
# dc1394_vloopback_SOURCES = dc1394_vloopback.c affine.c affine.h
# dc1394_vloopback_LDADD = -lm ../libdc1394/libdc1394_control.la -lraw1394
--- libdc1394-0.9.5/configure.in.lib64	2004-06-28 07:51:31.000000000 -0400
+++ libdc1394-0.9.5/configure.in	2004-09-20 05:29:38.350375931 -0400
@@ -8,6 +8,9 @@ AC_PROG_CC
 AC_PROG_LIBTOOL
 AM_PROG_LIBTOOL
 
+# We need X11 for examples
+AC_PATH_XTRA
+
 # Check for the new version of libraw1394 first since they might have both
 # installed and we want to use the newest version
 AC_CHECK_LIB(raw1394, raw1394_new_handle,
